src/config_server.h

```diff
@@ -0,0 +1,35 @@
+#ifndef NC_CONFIG_SERVER_H_
+#define NC_CONFIG_SERVER_H_
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#include <libyang/libyang.h>
+#include <stdint.h>
+#include <sys/types.h>
+
+#ifdef NC_ENABLED_TLS
+# include <openssl/x509.h>
```

Comment:
What is the include needed for? I see no X509 types used or anything else in this file.